apps/include/opt.h

```diff
@@ -185,6 +186,8 @@
             "Disallow session resumption on renegotiation"}, \
         {"no_legacy_server_connect", OPT_S_NOLEGACYCONN, '-', \
             "Disallow initial connection to servers that don't support RI"}, \
+        {"legacy_ssl_eof", OPT_S_LEGACY_SSL_EOF, '-', \
+            "Do not treat lack of close_notify as error"}, \
```

Comment:
Use legacy error handling if there is no close_notify

Reply:
Fixed. Thanks!